packages/devtools-source-map/src/source-map.js

```diff
@@ -11,6 +11,9 @@
 
 const { networkRequest } = require("devtools-utils");
 const { SourceMapConsumer, SourceMapGenerator } = require("source-map");
+SourceMapConsumer.initialize({
+  "lib/mappings.wasm": "https://unpkg.com/source-map@0.7.2/lib/mappings.wasm"
```

Comment:
It does work, I must say! The relevant documentation is [here](https://github.com/mozilla/source-map#sourcemapconsumerinitializeoptions). The relevant code is [here](https://github.com/mozilla/source-map/blob/43b2687e6e2f3cf6c22926aa473d5031fb7df89d/lib/read-wasm.js#L1).

I'm not sure what is the ideal way to do the wasm init context of a Webpack app. Would it work just to use the Node way of loading wasm? @fitzgen would love to get your input on this :)

Reply:
I don't know the answer here either, but this is going to need to be configurable so that this can also work in Mozilla Central.  There, we'll want to ship the wasm in Firefox and we'll need to use a resource URL or something.

Reply:
yep, we'll need to provide something when we start the worker...